javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -43,6 +43,7 @@ object Long {
   final val MIN_VALUE = -9223372036854775808L
   final val MAX_VALUE = 9223372036854775807L
   final val SIZE = 64
+  final val DIGIT_FIT_IN_INT = Array.tabulate[Int](37)(i => maxExpForRadix(i))
```

Comment:
`private lazy val DigitFitInInt = ...`
Also separated from the other (public) vals by a blank line.